client/index.js

```diff
@@ -22,7 +22,8 @@ const key = curry((type, payload) =>
 const sox = (args = {}) => {
   const {
     query = Function.prototype,
-    uri   = ''
+    uri   = '',
+    options = {}
```

Comment:
Since these are already options, maybe we should call this one `clientOptions`, or something else like that?

Reply:
definitely like `clientOptions` better.